apps/src/jigsaw/levels.js

```diff
@@ -402,12 +403,12 @@ module.exports = {
     isK1: true,
     image: {
       name: 'blocks',
-      width: 131,
-      height: 286,
+      width: 115,
```

Comment:
I guess that begs the question of what to do with these ones/how many constants to store...

Reply:
Yeah, even though there are a good amount of repeated values, there are still a bunch of exceptions. I think we'd need to maintain the ability to easily tweak a single level rather than all levels that happen to share a certain value currently. If we used a constant, it would probably need to be a map of level numbers to coordinates, which isn't so different from what we're doing now.